examples/snippets/uws/uws.py

```diff
@@ -203,7 +203,6 @@ class TAPParameters(Parameters):
 job_summary_xml = """
 <uws:job xmlns:uws="http://www.ivoa.net/xml/UWS/v1.0"
     xmlns:xlink="http://www.w3.org/1999/xlink"
-    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
```

Comment:
These namespaces have been removed as I've been mistakenly adding them up until now. They're the namespace that defines the _schema document_, not the instances.

Reply:
Ah, this is something that I am going to have to echo in the oai models, will include that in the next MR.